<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameScene</class>
 <widget class="QWidget" name="GameScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1916</width>
    <height>1397</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1916</width>
    <height>1397</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>1270</x>
     <y>260</y>
     <width>411</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI Light</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>AI 用时</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="recordTime">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI Light</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI Light</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>秒</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>1270</x>
     <y>140</y>
     <width>631</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI Light</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>难度：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="difficultyRecord">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI Light</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>简单</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="goBackBtn">
   <property name="geometry">
    <rect>
     <x>1570</x>
     <y>1250</y>
     <width>321</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回重新选择模式</string>
   </property>
  </widget>
  <widget class="QLabel" name="infoText">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>440</y>
     <width>581</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI Light</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameEndLabel">
   <property name="geometry">
    <rect>
     <x>1330</x>
     <y>580</y>
     <width>321</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正清刻本悦宋 简繁</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="meme">
   <property name="geometry">
    <rect>
     <x>1350</x>
     <y>760</y>
     <width>281</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正清刻本悦宋 简繁</family>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cheatBtn">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>1250</y>
     <width>261</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>悔棋</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
